pb/lmsysmd/dataset/v1/dataset.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package lmsysmd.dataset.v1;
+
+import "buf/validate/validate.proto";
+import "google/protobuf/timestamp.proto";
+
+service DatasetService {
+  rpc CreateDataset(CreateDatasetRequest) returns (CreateDatasetResponse) {
+    option idempotency_level = IDEMPOTENT;
+  }
+}
+
+message CreateDatasetRequest {
+  string url = 1 [ 
+    (buf.validate.field).required = true,
+    (buf.validate.field).string.uri = true 
+    ];
+}
+
+message CreateDatasetResponse {
+  repeated Dataset dataset = 1;
+}
+
+// Corresponds to one line on https://docs.google.com/spreadsheets/d/1-skT0JRJREJfn5qIQtSwyz2KVHMfLPJy0hOBfjyggZA/edit?gid=0#gid=0
+message Dataset {
+  string model = 1 [ (buf.validate.field).required = true ];
+  string vendor = 2 [ (buf.validate.field).required = true ] ;
+  uint32 param = 3 [ (buf.validate.field).required = true ] ;
+  google.protobuf.Timestamp model_date = 4 [ (buf.validate.field).required = true ] ;
```

Comment:
before applying this suggestion, is this the date that the model was released? or is this some other date? also consider using https://github.com/googleapis/googleapis/blob/master/google/type/date.proto if the precision is to the day.
```suggestion
  google.protobuf.Timestamp model_release_date = 4 [ (buf.validate.field).required = true ] ;
```

Reply:
Too be honest - reviewing the spreadsheet again, it is a little unclear to me what "model_date" refers to because there are multiple dates for the same model. I think maybe removing this field and adding a "creation_date" to the metadata field for the each dataset so we track how old the samples are.

Reply:
we could just clarify with the team about the model date. for create date that does not need to be exposed in the api? and it should be create time instead of create date if we are stamping it? wdyt

Reply:
yeah probably not necessary to expose the creation date in the api more for analysis. We could do create time but I don't think we need that level of precision for creation of the dataset.

Reply:
I'll remove the field then?

Reply:
> yeah probably not necessary to expose the creation date in the api more for analysis. We could do create time but I don't think we need that level of precision for creation of the dataset.

it's actually easier working with timestamps than date times, usually resorting to date times only when we want to obscure the precision so the client does not depend on the precision at all

Reply:
> I'll remove the field then?

i think we need to have it, just clarify with the lab team what it means. iirc it is model release date, which is a good field to filter by in any case

Reply:
I see - didn't know that. Cool then lets go with times!